lib/android.js

```diff
@@ -31,19 +33,35 @@ class JavaOnPathCheck extends DoctorCheck {
 
 // Check tools
 class AndroidToolCheck extends DoctorCheck {
-  constructor (toolName, toolPath) {
+  constructor () {
     super();
-    this.toolName = toolName;
-    this.toolPath = toolPath;
+    this.tools = ['adb', 'android', 'emulator', 'apkanalyzer'];
   }
 
   async diagnose () {
-    if (typeof process.env.ANDROID_HOME === 'undefined') {
-      return nok(`${this.toolName} could not be found because ANDROID_HOME is NOT set!`);
+    const listOfTools = this.tools.join(', ');
+    const sdkRoot = process.env.ANDROID_HOME || process.env.ANDROID_SDK_ROOT;
```

Comment:
this could also probably be exported from appium-adb

Reply:
Yea, I noticed it